### PR TITLE
Campaign lead boolean field value fix

### PR DESCRIPTION
**Pull request by escopecz**
Fix for https://github.com/mautic/mautic/issues/1300

### Testing
1. Make sure you have a boolean lead field. If not, create one.
2. Make a list with 2 or more leads.
3. Set the boolean field to true (or yes) to one of the leads.
4. Create a campaign which will test the boolean field condition. For example like the campaign below.
5. Run the `mautic:campaign:update` command to get the leads from the list to the campaign.
6. Run the `mautic:campaign:trigger` command to execute the campaign.

mautic-boolean-value-fix

The result is that all leads from the list will get the false tag. Doesn't matter what value was in the boolean field.

Apply this PR, remove the tags from the leads, clone the campaign and run the commands again. This time the leads should get the right tags. All other conditions has to work as well.

### CODE REVIEW

File: app/bundles/LeadBundle/Controller/AjaxController.php [hunk 1]
```diff
@@ -592,12 +592,19 @@ protected function updateLeadFieldValuesAction(Request $request)
         $choiceTypes = array('boolean', 'country', 'region', 'lookup', 'timezone', 'select', 'radio');
 
         if ($leadField && in_array($leadField->getType(), $choiceTypes)) {
-            $properties = $leadField->getProperties();
-
+            $properties = $field->getProperties();
```
Comment (alanhartless) [inline app/bundles/LeadBundle/Controller/AjaxController.php]: Should be `$leadField`

File: app/bundles/LeadBundle/Controller/AjaxController.php [hunk 2]
```diff
@@ -592,12 +592,19 @@ protected function updateLeadFieldValuesAction(Request $request)
...
+            $fieldType = $field->getType();
```
Comment (alanhartless) [inline app/bundles/LeadBundle/Controller/AjaxController.php]: Should be `$leadField`

**Comment by alanhartless** [comment]
The PR worked after I changed the two misnamed `$field` variables to `$leadField`.  Once that has been fixed, this'll be a go.

**Comment by escopecz** [comment]
You are right. How could I miss that... Fixed

**Comment by alanhartless** [comment]
:+1:

**Comment by vln104** [comment]
I could reproduce the issue and this PR fixed it for me, however, I retested by creating a new campaign with the same logic rather than cloning the previous campaign

**Comment by dbhurley** [comment]
+1 works as expected.

**Pull request merged**